src/components/look-controls.js

```diff
@@ -76,10 +77,15 @@ module.exports.Component = registerComponent('look-controls', {
       this.updateGrabCursor(data.enabled);
     }
 
-    // Reset pitch and yaw if disabling HMD.
-    if (oldData && !data.hmdEnabled && !oldData.hmdEnabled) {
-      this.pitchObject.rotation.set(0, 0, 0);
-      this.yawObject.rotation.set(0, 0, 0);
+    // Reset magic window eulers if tracking is disabled.
+    if (oldData && !data.magicWindowTrackingEnabled && oldData.magicWindowTrackingEnabled) {
```

Comment:
I think this should be 
```
if (oldData && !data.magicWindowTrackingEnabled && !oldData.magicWindowTrackingEnabled) 
{
...
}
```
equivalent to old code

Reply:
I think the old code might be wrong.

My intention is that it only resets when the property **changes** from disabled -> enabled.
With the old check it will only change the _second time_ that update is called with the value disabled.

So it will still work, but on the second `update` and will reset every update afterwards.

If I go back one more commit in git history of that line, it appears the original intent was closer to what I'm thinking with my change:
https://github.com/aframevr/aframe/blame/e9ba54ebcf724d828fba90fce9adc530e40526d6/src/components/look-controls.js#L45

`!hmdEnabled && oldData && hmdEnabled !== oldData.hmdEnabled` can be logically reduced to `!hmdEnabled && oldData && false !== oldData.hmdEnabled` and then to `!hmdEnabled && oldData && oldData.hmdEnabled`

Commit fe199d changed this: https://github.com/aframevr/aframe/commit/fe199da677e4fa5c5f83e3927fb8da39ca3ed766#diff-6285039eee70ed830f70f7d2f7b7dfa5R47

Reply:
Any thoughts on this? I can change it to be equivalent with the old code, even though I think the logic is wrong. It doesn't affect my use case of it anyway, so doesn't have to be my problem to deal with.

Reply:
Maybe we always want to reset if it value has changed?

`if (oldData && data.hmdEnabled !== oldData.hmdEnabled)`

Reply:
This is pretty close. What do you think about the comment above?

Reply:
Hey, sorry, have been quite busy lately. That makes sense. I think either way when enabling, the objects will almost immediately get overwritten from motion data, but I think it makes sense to always have a clean state when toggling.

Reply:
Actually, never mind. Just tested. And it's actually nice to _not_ reset when enabling motion controls. That way you can set your "starting point" for motion using touch.